src/models/projects.models.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+
+const projectSchema = mongoose.Schema(
+  {
+	id: {
+		type: mongoose.Schema.Types.ObjectId,
+		required: [true],
+	},
+	name: {
+		type: String,
+		required: true,
+	},
+	customerId: {
+    	type: mongoose.Schema.Types.ObjectId,
+	  	required: true,
+	},
+	quotationId: {
```

Comment:
QuotationId is not present in Project schema, b'se we have added projectId in quotation Schema, But even if we want to keep quotationIds of a project then we have to keep array of quotationId b;'se a single Project can have multiple quotations inside it and you have to update the mongo query to get proejct accordingly.